index.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link href="./styles.css" rel="stylesheet">
+  <title>English courses</title>
+</head>
+<body class="container">
```

Comment:
- [ ] If you check your page carefully, you will realize a white margin all around your page which is not supposed to be there, so please work on this issue to try to fix it.